lib/containers/utils.pm

```diff
@@ -252,11 +252,18 @@ sub basic_container_tests {
     assert_script_run "$runtime run --rm --init $tumbleweed ps --no-headers -xo 'pid args' | grep '1 .*init'";
 
     if (script_run('command -v man') == 0) {
-        $ret = script_run("man -P cat $runtime build | grep '$runtime-build - Build'");
-        if ($ret != 0) {
-            record_soft_failure("bsc#1215465 - man -P cat docker build fails") if (is_tumbleweed || is microos);
-        } else {
-            die("man -P cat docker build fails");
+        my $output = script_output("man -P cat $runtime build", sub { m/$runtime-build - Build/ }, proceed_on_failure => 1);
```

Comment:
don't you mean validate_script_output?

Reply:
No, because this command can fail due to bsc#121546, see the next block.

`validate_script_output` would fail when `man -P cat` fails, so this would break the test run when it should softfail.

Reply:
I haven't seen in the interface of `script_output` that it would handle a code as argument. Where have you found it?

Reply:
Orrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrr

Yes that is wrong.

Reply:
Amended, please check again.